tests/unit/test-class-wc-payment-gateway-wcpay-subscriptions.php

```diff
@@ -512,6 +512,29 @@ public function test_render_custom_payment_meta_input_invalid_value() {
 		$this->wcpay_gateway->render_custom_payment_meta_input( $subscription, 'field_id', 'invalid_value' );
 	}
 
+	public function test_render_custom_payment_meta_input_many_tokens() {
```

Comment:
I'd suggest using `multiple` instead of `many` in the test name.

Reply:
Changed it to `multiple` . Thanks!